laws-africa/peachjam
# Show that amendments are being applied to a work

This PR adds logic to show that a work has outstanding amendments yet to be applied.

To test locally:
- Add an extra `work_amendment` item to the list of  `work_amendments` in the metadata_json for the document. Make sure that the date for new `work_amendment` comes after the latest work amendment date. This date will be used in coming up with the relevant notice.




![image](https://user-images.githubusercontent.com/15012985/205963396-02bc2d8d-be63-4300-86ab-3eeb1baf6a92.png)
